### ISSUE TITLE
sh: 90: local: 2.22.1: bad variable name

### ISSUE THREAD

**Issue by kzhui125** [joseluisq/static-web-server#273]
**Describe the bug**

**To Reproduce**

```sh
curl --proto '=https' --tlsv1.2 -sSfL https://get.static-web-server.net | sh
```

**Logs/Screenshots**
If applicable, add logs or screenshots to help explain your problem.

<img width="527" alt="image" src="https://github.com/static-web-server/static-web-server/assets/11925432/b43fb429-9e8a-4c6a-a34a-ba54685ff511">

**Environment and Specs**
 - static-web-server: version from get.static-web-server.net
 - OS: Linux 4.19.0-16-amd64 x86_64

**Comment by joseluisq** [comment]
@kzhui125 can you please test the bugfix https://github.com/static-web-server/static-web-server/pull/274 in your Linux? Use the following command:

`curl --proto '=https' --tlsv1.2 -sSfL https://raw.githubusercontent.com/static-web-server/static-web-server/fix-installer-script/scripts/installer.sh | sh`

**Comment by joseluisq** [comment]
Resolved in [v2.23.0](https://github.com/static-web-server/static-web-server/releases/tag/v2.23.0)
